packaging/os/yum.py

```diff
@@ -882,10 +883,11 @@ def main():
     if params['install_repoquery'] and not repoquery and not module.check_mode:
```

Comment:
`repoquery` doesn't exist anymore, so you should remove that part of the condition.